src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -9,16 +11,19 @@
 
 public class Parser {
 
+    public static final String MESSAGE_INVALID_ID = "Please enter a valid numerical index!";
+    public static final String MESSAGE_EMPTY_INPUT = "Use /help for a list of supported commands!";
```

Comment:
Instead of making these variables public, I suggest using a `final class` to store your constants to reduce code clutter. The commands can also be stored in `seedu.duke.constants` so that they can be reused in testing without needing magic strings.

https://www.baeldung.com/java-constants-good-practices#2-constants-class

Reply:
@jinxuan-owyong Noted. Have made the amendments as per your suggestion.